state-synchronizer/src/coordinator.rs

```diff
@@ -62,6 +63,8 @@ pub(crate) enum CoordinatorMessage {
     Commit(
         // committed transactions
         Vec<Transaction>,
+        // reconfiguration events
```

Comment:
Btw, it's guaranteed that they're ordered in the ascending order of their sequence numbers.